fonts.css

```diff
@@ -1,27 +1,31 @@
 @font-face {
-  font-family: 'ptserif';
-  src: url('ptserif-bold-webfont.woff') format('woff');
+  font-family: 'PTSerif';
+  src: url('fonts/PTSerifBold.woff2') format('woff2'), 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6240796)